x-pack/plugins/security_solution/public/management/pages/policy/view/trusted_apps/layout/policy_trusted_apps_layout.tsx

```diff
@@ -114,7 +115,7 @@ export const PolicyTrustedAppsLayout = React.memo(() => {
 
   return policyItem ? (
     <div>
-      {!displaysEmptyStateIsLoading && !displaysEmptyState ? (
+      {!hasAssignedTrustedApps.loading && !displaysEmptyState ? (
```

Comment:
**Note:** This is very odd to read but I guess one of the simplified sub-clauses in here is truthy sometimes. `!hasAssignedTrustedApps.loading && hasAssignedTrustedApps.loading` would never be true though, and that's odd to see here.

Reply:
Suggestion: Maybe not use the ternary operator to not have to render `null` when the condition is not true.

Reply:
Agree the condition was odd, will change it. About render null or not, this header needs to be displayed only when there is no empty state displayed

Reply:
done in: https://github.com/elastic/kibana/pull/119290/commits/a54edb0c89e6c278fb0666cb1d9ba719630619ce